Bottom: 8df3042396aae5087e4abb402ad4ea326ee03ad6
Top:    c0315433c0cb3da26eba8654fefa05f80694cf18
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-25 13:44:04 -0500

Update python docker client library

Python's docker-py client library has updated to 1.10.6 at this time. This
was the last version in the 1.x series. Newer versions are under the
package name 'docker' on pypi.


---
diff --git a/roles/docker-common/tasks/packages.yml b/roles/docker-common/tasks/packages.yml
index 16f732c..ca85796 100644
--- a/roles/docker-common/tasks/packages.yml
+++ b/roles/docker-common/tasks/packages.yml
@@ -1,18 +1,6 @@
 ---
 # file: roles/docker-common/tasks/package.yaml
 
-- name: install yum package docker-python (1.4.0)
-  yum:
-    name: python-docker-py-1.4.0
-    state: present
-  when: ansible_os_family == "RedHat"
-
-- name: Install docker-py (1.4.0)
-  command: pip2 install docker-py==1.4.0
-  when: ansible_os_family == "Archlinux"
-
-- name: Install docker-py (1.4.0)
+- name: Install docker python module
   pip:
-    name: docker-py
-    version: 1.4.0
-  when: (ansible_os_family != "RedHat") and (ansible_os_family != "Archlinux")
+    name: docker==2.7.0
